vividus-engine/src/main/java/org/vividus/steps/ExpressionAdaptor.java

```diff
@@ -74,6 +76,10 @@ public ExpressionAdaptor(StoryControls storyControls)
      */
     public Object processRawExpression(String expression)
     {
+        if (expression.equals(NULL_EXPRESSION))
```

Comment:
so the nested null expression is not supported? I believe there is not much of a sense in the nested #{null} expression, but since typical nested expressions are nested by design we should disclose this behaviour

Reply:
> so the nested null expression is not supported?

I agree it doesn't make much sense


> but since typical nested expressions are nested by design we should disclose this behaviour

👍 please add note to docs